library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayLifecycleCallback.kt

```diff
@@ -41,25 +41,25 @@ internal class SessionReplayLifecycleCallback(
         TimeUnit.MILLISECONDS,
         LinkedBlockingDeque()
     )
-    internal var recorder: Recorder = ScreenRecorder(
-        RecordedDataProcessor(
-            rumContextProvider,
-            timeProvider,
-            processorExecutorService,
-            recordWriter,
-            recordCallback
-        ),
-        SnapshotProducer(privacy.mapper()),
-        timeProvider
+    val processor = RecordedDataProcessor(
```

Comment:
can be `internal` it seems.

```suggestion
    internal val processor = RecordedDataProcessor(
```